### PR TITLE
Return error if test fails

### PR DESCRIPTION
**Pull request by Amir-61**
Return error if test fails

/to: @superkhau

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ describe('mixin example', function() {
   it('should be assigned `squirrel` before saving', function(done) {
     var Dog = app.models.Dog;
     Dog.create({breed: 'Pitbull'}, function(err, dog) {
+      if(err) return done(err);
```
Comment (superkhau) [inline test/test.js]: space in front of if

**Comment by superkhau** [comment]
LGTM, make that change and :shipit:! ;)

**Comment by superkhau** [comment]
You should update this project to use ESLint btw, it would've caught the error before review.

**Comment by Amir-61** [comment]
> You should update this project to use ESLint btw, it would've caught the error before review.

I created a [GH issue](https://github.com/strongloop/loopback-example-mixins/issues/7) for that which is assigned to me; I'll take care of it soon :-)

**Pull request merged**